homeassistant/components/ai_speaker/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""Support for AI-Speaker."""
+import logging
+
+from aisapi.ws import AisWebService
+
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN
+
+PLATFORMS = ["sensor", "media_player"]
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass, config):
+    """Set up the initial domain configuration."""
+
+    async def async_command(service):
+        """Publish command to AI-Speaker WS."""
+        web_session = aiohttp_client.async_get_clientsession(hass)
+        ais_url = service.data["ais_url"]
+        ais_gate = AisWebService(hass.loop, web_session, ais_url)
```

Comment:
Please create the web service instance in `async_setup_entry`. Store the instance in `hass.data`.

Reply:
done in new commit @asystentka-jolka
improvements after first review
6214ef8